herald.py

```diff
@@ -129,26 +131,41 @@ async def runHerald(ctx, args):
         return
 
     pickle.dump(heraldDict, open("herald/heraldUsers.p", "wb"))
+    return
 
 async def playHerald(member):
 
     # Only run if user has selected audio
     if member.id in heraldDict.keys():
 
-        print(f'{member.name} has played Herald!')
+        current_time = datetime.datetime.now()
+
+        #only run if user has never joined voice channel or the time between joining exceeds delay
+        if (isinstance(heraldDict[member.id], type(None))) or ((current_time - heraldDict[member.id].lastUseTime).total_seconds() > wait_timer):
```

Comment:
Should the isInstance compare heraldDict[member.id].lastUseTime

Reply:
Oops, yes. Good catch.